shop.html

```diff
@@ -0,0 +1,599 @@
+            >
+              More Travel Tech &rarr;
+            </button>
+          </div>
+        </div>
+        <div>
+          <div class="sm:flex sm:items-baseline sm:justify-between">
+            <h2 class="text-2xl font-bold">Accessories</h2>
+          </div>
+          <div
+            class="mt-6 grid grid-cols-1 gap-y-10 gap-6 md:grid-cols-2 lg:grid-cols-3 lg:gap-x-8"
+          >
+            <div class="group relative">
+              <a href="#">
+                <div
+                  class="h-96 w-full overflow-hidden rounded-lg sm:aspect-h-3 sm:aspect-w-2 group-hover:opacity-75 sm:h-auto"
+                >
+                  <img
+                    src="/assets/images/travelpillow.jpg"
+                    alt=""
+                    class="w-full h-96 object-cover object-center"
+                  />
+                </div>
+                <h3 class="mt-4 text-travel-orange text-lg font-semibold">
+                  Airtags
+                </h3>
+              </a>
+              <p class="mt-1 text-sm">
+                Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum
+              </p>
+            </div>
+            <div class="group relative">
+              <a href="#">
+                <div
+                  class="h-96 w-full overflow-hidden rounded-lg sm:aspect-h-3 sm:aspect-w-2 group-hover:opacity-75 sm:h-auto"
+                >
+                  <img
+                    src="/assets/images/traveldocs.jpg"
+                    alt=""
+                    class="w-full h-96 object-cover object-center"
+                  />
+                </div>
+                <h3 class="mt-4 text-travel-orange text-lg font-semibold">
+                  Travel Documents Holder
+                </h3>
+              </a>
+              <p class="mt-1 text-sm">
+                Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum
+              </p>
+            </div>
+            <div class="group relative">
+              <a href="#">
+                <div
+                  class="h-96 w-full overflow-hidden rounded-lg sm:aspect-h-3 sm:aspect-w-2 group-hover:opacity-75 sm:h-auto"
+                >
+                  <img
+                    src="/assets/images/Passport holder.jpg"
+                    alt=""
+                    class="w-full h-96 object-cover object-center"
+                  />
+                </div>
+                <h3 class="mt-4 text-travel-orange text-lg font-semibold">
+                  Passport Holder
+                </h3>
+              </a>
+              <p class="mt-1 text-sm">
+                Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum
+              </p>
+            </div>
+          </div>
+          <div class="flex justify-end items-center mt-4">
+            <button
+              class="text-travel-orange font-semibold hover:text-orange-600"
+            >
+              More Accessories &rarr;
```

Comment:
Missing link on button

Reply:
Leaving as a dummy button as no link to a different page for all products